src/app/styles/components/_alerts.scss

```diff
@@ -0,0 +1,23 @@
+.alert {
+    padding: 1rem;
+    text-align: center;
+
+    .alert-title {
+        position: relative;
+        margin-top: 0;
+        margin-bottom: 1rem;
+    }
+
+    .close {
+        position: absolute;
+        top: 0;
+        right: 0;
+        background: none;
+        border: none;
+        cursor: pointer;
+    }
+}
+
+.alert-bicycle {
+    background-color: #F0A438;
```

Comment:
Colors are defined as variables in a separate SASS colors file; I'm guessing this is one of the existing colors there.